src/wallet/rpcwallet.cpp

```diff
@@ -2344,8 +2344,14 @@ static RPCHelpMan getbalances()
 {
     return RPCHelpMan{
         "getbalances",
-        "Returns an object with all balances in " + CURRENCY_UNIT + ".\n",
-        {},
+        "Returns an object with all balances in " + CURRENCY_UNIT + ", optionally including the balance change given one or more specified transactions.\n",
+        {
+            {"rawtxs", RPCArg::Type::ARR, RPCArg::Optional::OMITTED_NAMED_ARG, "An array of hex strings of raw transactions.\n",
```

Comment:
It seems weird to have an array of raw transactions as a positional argument here.